docs-conceptual/azps-2.0.0/migrate-az-1.0.0.md

```diff
@@ -42,146 +47,181 @@ This document describes the changes between the 6.x versions of AzureRM and Az v
+| Azure.AnalysisServices | Az.AnalysisServices | Yes |
+| AzureRM.Profile | Az.Accounts | Yes |
+| AzureRM.Insights | Az.Monitor | Yes |
+| AzureRM.DataFactories | Az.DataFactory | Yes |
+| AzureRM.DataFactoryV2 | Az.DataFactory | Yes |
+| AzureRM.RecoveryServices.Backup | Az.RecoveryServices | No |
+| AzureRM.RecoveryServices.SiteRecovery | Az.RecoveryServices | No |
+| AzureRM.Tags | Az.Resources | No |
+| AzureRM.MachineLearningCompute | Az.MachineLearning | No |
+| AzureRM.UsageAggregates | Az.Billing | No |
+| AzureRM.Consumption | Az.Billing | No |
 
-#### Migrating #Requires Statements
-Scripts that use #Requires to declare a dependency on AzureRM modules should be updated to use the new module names
-```powershell
+The changes in module names mean that any script that uses ```#Requires``` or ```Import-Module``` to load specific modules will need to be changed to use the new module instead. For modules where the cmdlet suffix has not changed,
+this means that although the module name has changed, the suffix indicating the operation space has _not_.
+
+#### Migrating #Requires and Import-Module Statements
+
+Scripts that use `#Requires` or `Import-Module` to declare a dependency on AzureRM modules must be updated to use the new module names. For example:
+
+```azurepowershell-interactive
 #Requires -Module AzureRM.Compute
 ```
 
-Should be changed to
-```powershell
+Should be changed to:
+
+```azurepowershell-interactive
 #Requires -Module Az.Compute
 ```
 
-#### Migrating Import-Module Statements
-Scripts that use ```Import-Module``` to load AzureRM modules will need to be updated to reflect the new module names.
-```powershell
+For `Import-Module`:
+
+```azurepowershell-interactive
 Import-Module -Name AzureRM.Compute
 ```
 
-Should be changed to
-```powershell
+Should be changed to:
+
+```azurepowershell-interactive
 Import-Module -Name Az.Compute
 ```
 
 ### Migrating Fully-Qualified Cmdlet Invocations
-Scripts that use module-qualified cmdlet invocations, like
-```powershell
+
+Scripts that use module-qualified cmdlet invocations, such as:
+
+```azurepowershell-interactive
 AzureRM.Compute\Get-AzureRmVM
 ```
 
-Should be changed to use the new module and cmdlet names
-```powershell
+Must be changed to use the new module and cmdlet names:
+
+```azurepowershell-interactive
 Az.Compute\Get-AzVM
 ```
 
-### Migrating Module Manifest Dependencies
-Modules that express dependencies on AzureRM modules through a module manifest (.psd1) file will need to updated the module names in their 'RequiredModules' section
+### Migrating module manifest dependencies
+
+Modules that express dependencies on AzureRM modules through a module manifest (.psd1) file will need to updated the module names in their `RequiredModules` section:
 
-```powershell
+```azurepowershell-interactive
 RequiredModules = @(@{ModuleName="AzureRM.Profile"; ModuleVersion="5.8.2"})
 ```
 
-Should be changed to
+Must be changed to:
 
-```powershell
+```azurepowershell-interactive
```

Comment:
This is not an executable statement. It is a member of a hash table in the manifest file. So the code fence type should not be interactive. It should be "powershell".